RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevicesFaceToFace.json

```diff
@@ -0,0 +1,901 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "RespiratoryAssistDevicesFaceToFace",
+  "name": "RespiratoryAssistDevicesFaceToFace",
+  "title": "Respiratory Assist Device Face-to-Face (F2F) Encounter",
+  "status": "draft",
+  "date": "2020-06-16",
+  "publisher": "Da Vinci DTR",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+      "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+    ]
+  },
+  "subjectType": ["Patient"],
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/RespiratoryAssistDevicesFaceToFace-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicClinicalInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
```

Comment:
We should not need to include BasicPatient and BasicPractitioner library anymore.